hydra.gemspec

```diff
@@ -35,5 +35,5 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'ldp', '~> 0.3.0'
   gem.add_dependency 'active-triples', '~> 0.6.0'
   gem.add_dependency 'nom-xml', '~> 0.5.1'
-  gem.add_development_dependency 'github_api', '~> 0.10.1'
+  gem.add_development_dependency 'github_api', '~> 0.10'
```

Comment:
should this be `~> 0.12` now?  0.10.1 meets the current spec requirements, but obviously doesn't work.